cmd/beekeeper/cmd/simulate.go

```diff
@@ -26,8 +26,21 @@ func (c *command) initSimulateCmd() (err error) {
 
 	cmd := &cobra.Command{
 		Use:   "simulate",
-		Short: "runs simulations on a Bee cluster",
-		Long:  `Runs simulations on a Bee cluster.`,
+		Short: "Runs simulations on a Bee cluster",
```

Comment:
deprecated, we are not using simulations anymore.

Reply:
Do not remove simulations. Remove Long description. For Short use:
```
Short: "Runs simulations on a Bee cluster. Deprecated",
```
Or something like that.